core/testing/src/main/java/io/github/droidkaigi/confsched2023/testing/robot/SearchScreenRobot.kt

```diff
@@ -0,0 +1,101 @@
+package io.github.droidkaigi.confsched2023.testing.robot
+
+import androidx.compose.ui.test.hasTestTag
+import androidx.compose.ui.test.isRoot
+import androidx.compose.ui.test.junit4.AndroidComposeTestRule
+import androidx.compose.ui.test.onFirst
+import androidx.compose.ui.test.onLast
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.performTouchInput
+import androidx.compose.ui.test.swipeLeft
+import com.github.takahirom.roborazzi.captureRoboImage
+import io.github.droidkaigi.confsched2023.designsystem.theme.KaigiTheme
+import io.github.droidkaigi.confsched2023.sessions.SearchScreen
+import io.github.droidkaigi.confsched2023.sessions.SearchScreenTestTag
+import io.github.droidkaigi.confsched2023.sessions.component.FilterChipWithDropdownMenuItemTestTag
+import io.github.droidkaigi.confsched2023.sessions.component.SearchFilterTestTag
+import io.github.droidkaigi.confsched2023.testing.RobotTestRule
+import io.github.droidkaigi.confsched2023.testing.coroutines.runTestWithLogging
+import kotlinx.coroutines.test.TestDispatcher
+import javax.inject.Inject
+import kotlin.time.Duration.Companion.seconds
+
+class SearchScreenRobot @Inject constructor(
```

Comment:
I implemented it with reference to `TimetableScreenRobot`.

Reply:
Wow, great!